cmd/conduit/main.go

```diff
@@ -117,6 +117,8 @@ func makeConduitCmd() *cobra.Command {
 			return runConduitCmdWithConfig(cfg)
 		},
 		SilenceUsage: true,
+		// Silence errors because our logger will catch and print any errors
+		SilenceErrors: true,
```

Comment:
Whats the difference between this and changing from `RunE` to `Run`?

Reply:
I think the big difference is that we don't need to call `os.Exit(1)` inside the command function.  Instead we can return an error (which causes defer'd functions to run) and return an appropriate error code to the command line at the main function level.